### DISCUSSION TITLE
Contribution to Open Issues

### DISCUSSION THREAD

**Discussion by lishakothari** [freeCodeCamp/Developer_Quiz_Site#664] [General]
Hi there folks! @jdwilkin4 
What's the best practice to contribute? Do we send in different PRs for each question or different PRs per category. 
Looking forward to add more questions & help fix some bugs

**Comment by jdwilkin4** [comment]
Hi @lishakothari ! 

For bugs in the codebase, it is best to open an issue first detailing the problem.
If you are contributing multiple quiz questions at once, then you can open just one PR for that. 

Here is the link to the complete [contributing guide](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
